<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistWindow</class>
 <widget class="QWidget" name="PlaylistWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>554</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>4</number>
   </property>
   <item row="4" column="0">
    <widget class="Playlist::BottomBar" name="bottomBar" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labTotalTime">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string notr="true">0m 0s</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Icons/broom.png</normaloff>:/Icons/broom.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Gui::MenuToolButton" name="toolButton">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Playlist::TabWidget" name="twPlaylists">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="tabsClosable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab_pl1">
      <attribute name="title">
       <string notr="true">+</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::MenuToolButton</class>
   <extends>QPushButton</extends>
   <header>Gui/Utils/MenuTool/MenuToolButton.h</header>
  </customwidget>
  <customwidget>
   <class>Playlist::TabWidget</class>
   <extends>QTabWidget</extends>
   <header>Gui/Playlist/PlaylistTabWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Playlist::BottomBar</class>
   <extends>QWidget</extends>
   <header>Gui/Playlist/PlaylistBottomBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
